examples_new/optimization/scipy/optimize_power_density_1D.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2020 NREL
+
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
```

Comment:
I would add some description of what is going on in this example...

Reply:
Good point. I added an explanation of the intent of the example.